platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -57,6 +57,7 @@
  * @param runningHashUpdateScheduler                the scheduler for the running hash updater
  * @param futureEventBufferScheduler                the scheduler for the future event buffer
  * @param issDetectorScheduler                      the scheduler for the iss detector
+ * @param hashLoggerScheduler                      the scheduler for the signed state logger
```

Comment:
this javadoc is misaligned with the rest of the block

Reply:
is this something the linter could have caught?

Reply:
Unfortunately, our linter isn't smart enough to catch these alignment issues :(

Reply:
Unresolving this so I can leave a comment. Feel free to resolve after you read it.

There is a way to get intellij to automatically format your javadocs for you. Ping me some time and I can help you figure it out.